docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -100,10 +97,10 @@ The following UI components are highly recommended for channels.
 
 ## Documentation
 
-| Link                                                                                                                                    | Description                                                                 |
```

Comment:
I will defer to @aglensmith to comment on how the tables are formatted in this article.